src/main/resources/public/raml/common-schema.raml

```diff
@@ -61,6 +61,12 @@ subscription: |
           },
           "userName":{
              "type":"string"
+          },
```

Comment:
remove restUser and token. Front end will have to add it to formValue in notificationMessageKeyValues

Reply:
Done